modules/_canvas-kit-react/index.ts

```diff
@@ -9,7 +9,6 @@ export * from '@workday/canvas-kit-react-color-picker';
 export * from '@workday/canvas-kit-react-common';
```

Comment:
Things moved from labs should also be removed from the universal module's `package.json`